src/commands/help.rs

```diff
@@ -1,35 +1,26 @@
 use super::super::directory::scan_directory;
-use std::{io, iter::Peekable, slice::Iter};
+use std::io;
 
 macro_rules! help_template {
     () => {
-        r#"echo -e
-'This is an instance of tome, running against the directory {}.
-\nThe commands are namespaced by the directory structure.
```

Comment:
I simplified the escaping by printing this directly via eprintln vs trying to echo it.